digitalpalireader/content/js/format.js

```diff
@@ -560,6 +560,7 @@ function clearDivs(which) { // place divs to be cleared here
 
 function makeToolbox(main,aux,title,conv,ex,save,trans) {
 	if (DPR_PAL.isWeb) {
+    $('#nav-context-div').html('<label class="font-weight-bold">Context Menu</label><div id="tbOpener" class="tiny">&there4;</div><div id="tbContainer2"><div id="MainToolbar" class="obutc">'+main+'</div>'+(aux?'<div id="auxToolbar" class="obutc">'+aux+'</div>':'')+'</div>');
```

Comment:
please move this to the markup.

we will have to make this contextual - that can be done later along with other contextual stuff.